### DISCUSSION TITLE
Synch Normalized Database with Denormalized Database

### DISCUSSION THREAD

**Discussion by bwscheller** [Mimetis/Dotmim.Sync#706] [Q&A]
Currently have a SQL Server database that is used for transactions where the data is migrated to a data warehouse on an hourly basis.  The data warehouse contains the majority of the same information, no soft deleted records, but is in a denormalized form for reporting.  Can Dotmim.Sync be utilized to synchronize between different data structures?

If so, what component would need to be customized, Orchestrator?

Thanks.

Brian Scheller

**Comment by Mimetis** [comment]
I'm sorry, the client databases and the server database should share the exact same schema
